src/components/Filter.js

```diff
@@ -39,8 +39,9 @@ const SubmitOnEnter = ({ children }) => {
 
 export default ({ children, defaultValues, pageSize, sorts, transform }) => {
   const [urlState, updateUrl] = useUrlParamState()
+  const process = useCallback(values => updateUrl({ ...values, pageSize: urlState.pageSize, sort: urlState.sort }))
```

Comment:
This looks to be missing the deps list, also reformat like this for easier reading:
```
const process = useCallback(values => {
  updateUrl({ ...values, pageSize: urlState.pageSize, sort: urlState.sort }))
}, [urlState, updateUrl])
```